### DISCUSSION TITLE
Welcome to Koriki for RG35XX Discussions!

### DISCUSSION THREAD

**Discussion by acmeplus** [rg35xx-cfw/Koriki#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KennyL0gg1ns** [comment]
So here is my feedback for this iteration:

- B button can't be rebound because it's also cancel in the menus for DS games, so it backs out of the key config menu when you try to assign it

- Performance is mostly good, but sometimes tanks

- Switching screens needs to be simplified, and a simple picture in picture mode, toggled through R2 or L2 as these are the only 2 buttons left to use



Options to assign this stuff would make this perfect.
